sdk/firstload/src/index.tsx

```diff
@@ -361,6 +368,9 @@ const H: HighlightPublicInterface = {
 	},
 	metrics: (metrics: Metric[]) => {
 		try {
+			for (const m of metrics) {
+				H.gauge(m)
+			}
```

Comment:
Doesn't `recordMetric` already do this in its implementation? Would this result in duplicate metrics since we're calling `gauge` here and inside `recordMetric`?

Reply:
good catch!